apptemplates/__init__.py

```diff
@@ -39,12 +39,15 @@ def get_app_template_dir(app_name):
 
 
 if django.VERSION[:2] >= (1, 9):
-    def get_template_path(template_dir, template_name):
+    def get_template_path(template_dir, template_name, loader=None):
         """Return Origin object with template file path"""
-        return Origin(name=join(template_dir, template_name))
+        return Origin(name=join(template_dir, template_name),
+                      template_name=template_name,
+                      loader=loader)
 else:
-    def get_template_path(template_dir, template_name):
+    def get_template_path(template_dir, template_name, loader=None):
         """Return template file path (for Django < 1.9)"""
+        _ = loader  # noqa

```

Comment:
Remove this line (you silence the PyLint warning implicitly and the flake8 warning explicitly, which is unneeded), and instead make the function signature look like this to silence the PyLint waning explicitly:
``` python
    def get_template_path(template_dir, template_name,
                          loader=None):  # pylint: disable=unused-argument
```